lang/talon/talon_common.talon

```diff
@@ -0,0 +1,22 @@
+#Defines commands common to both python and talon files
+tag: user.talon
+tag: user.talon_python
+-
+
+tag set [{user.talon_tags}]:
+    tag = talon_tags or ""
+    user.talon_code_enable_tag(tag)
+
+# requires user.talon_populate_lists tag. do not use with dragon
+list {user.talon_lists}: "{{{talon_lists}}}"
+# requires user.talon_populate_lists tag. do not use with dragon
+capture {user.talon_captures}: "<{talon_captures}>"
+
+setting {user.talon_settings}: user.talon_code_enable_setting(talon_settings)
+
+#commands for dictating key combos
+key <user.keys> over: "{keys}"
+key <user.modifiers> over: "{modifiers}"
+
+action {user.talon_actions}:
+    user.talon_code_insert_function(talon_actions, edit.selected_text())
```

Comment:
This implementation is the same as code_insert_function so there is no need for an additional action to be defined here.

Reply:
I don't think this is correct.

The intent here is to able to use "action" command for talon-specific action calls and append "actions." as required in python and omit in .talon files.

The existing code_insert_function can then can be used for regular python functions even when the talon_python tag is active. 

I think this necessitates a separate action unless I'm missing something

Reply:
Gotcha, I see now — @AndreasArvidsson and I didn't realize the the same action would be used across .talon/.py files. We had only thought of the first one. However it should still be possible to implement the action in terms of the existing user.code_insert_function, right? Just trying to eliminate duplicate implementations.

Reply:
As I recall, reusing code_insert_function in the python case by inserting "actions." before calling down creates an additional entry on the undo stack, which I felt is undesirable. I didn't see a better way.

In the .talon file case, I suppose we can do so without issue.